lib/tasks/migration.rake

```diff
@@ -76,4 +78,18 @@ namespace :migration do
     # mail.deliver_now
   end
 
+  desc "Set the new last_update based on the updated_at before the migration"
+  task cards_last_update: :environment do
+    CSV.foreach("#{Rails.root}/lib/tasks/cards_updated_at.csv", :headers => true) do |row|
+        card = Card.find_by("id = ?", row[0])
```

Comment:
```suggestion
        card = Card.find(row[0])
```

Reply:
I'm using `find_by` because it returns nil if not found, as opposed to `find` that throws an error, which is not what we want there.

Reply:
```suggestion
        card = Card.find_by(id: row[0])
```